.github/scripts/backup_qdrant.py

```
@@ -0,0 +1,158 @@
+    try:
+        os.makedirs(SNAPSHOTS_DIR, exist_ok=True)
+        print('creating snapshots directory')
+        for snapshot in snapshot_list:
+            snapshot_url = snapshot.get("snapshot_url")
+            print("snapshot_url: ")
+            print(snapshot_url)
+            print('downloading snapshot: ', snapshot_url)
+            snapshot_name = os.path.basename(snapshot.get("snapshot_info").name)
+            local_snapshot_path = os.path.join(SNAPSHOTS_DIR, snapshot_name)
+
+            response = requests.get(
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)